pontoon/settings/base.py

```diff
@@ -1127,13 +1130,20 @@ def account_username(user):
 except ValueError as e:
     raise ValueError(f"Error: {e}")
 
-# Used for Review Master and Translation Champion
+# Used for Translation Champion
 BADGES_TRANSLATION_THRESHOLDS = list(
     map(
         int,
         os.environ.get("BADGES_TRANSLATION_THRESHOLDS", "5, 50, 250, 1000").split(","),
     )
 )
+# Used for Review Master
+BADGES_REVIEW_THRESHOLDS = list(
+    map(
+        int,
+        os.environ.get("BADGES_REVIEW_THRESHOLDS", "7, 75, 375, 1500").split(","),
```

Comment:
Please add a comment explaining why are the numbers higher than in the Translation Threshold.